fabric-common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fabric-common",
-  "version": "1.4.11",
+  "version": "1.4.12-snapshot",
```

Comment:
I think there is not any concrete code in fabric-common, should we keep it private instead of publish an empty library to npm?